src/Components/EditLogForm.js

```diff
@@ -0,0 +1,98 @@
+import { useState, useEffect } from "react";
+import { useParams, Link, useNavigate } from "react-router-dom";
+import axios from "axios";
+
+export default function EditLogForm() {
+  let { index } = useParams();
+
+  const [log, setLog] = useState({
+    captainName: "",
+    title: "",
+    post: "",
+    mistakesWereMadeToday: "false",
+    daysSinceLastCrisis: "",
+  });
+  let navigate = useNavigate();
+  const handleTextChange = (event) => {
+    setLog({ ...log, [event.target.id]: event.target.value });
+  };
+
+  const handleCheckboxChange = () => {
+    setLog({ ...log, mistakesWereMadeToday: !log.mistakesWereMadeToday });
+  };
+
+  useEffect(() => {
+    axios
+      .get(`${process.env.REACT_APP_API_URL}/bookmarks/${index}`)
+      .then((response) => {
+        setLog(response.data);
+      })
+      .catch((e) => {
+        console.log(e);
+      });
+  }, [index]);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    axios
```

Comment:
If you are editing a log, what http very should you be using here instead of a `get`?